packages/env-options/README.md

```diff
@@ -0,0 +1,75 @@
+# Parameterizing Modules with Environment Options
+
+JavaScript module semantics resists attempts to parameterize a module's
```

Comment:
Semantics is indeed a singular, but only because English adopted Greek -ikos suffix inconsistently (-ic or -ics) and we should be more angry about it.

Reply:
The principled way to express our anger is to deliberately spell it “semantology” but this is not the time or place 😉

Reply:
So should this be `resist` or `resists`?

Reply:
Unresolving

Reply:
`semantics resist` is correct.

Reply:
Cool. Reresolving.